### ISSUE TITLE
Wish:  Tell which page was not found

### ISSUE THREAD

**Issue by stappersg** [joseluisq/static-web-server#108]
**Is your feature request related to a problem? Please describe.**
Would like to known which pages the visitors trying to reach.

**Describe the solution you'd like**
That
```text
2022-05-08T13:40:55.500059Z  WARN static_web_server::error_page: method=GET status=404 error=404
2022-05-08T13:40:55.547381Z  WARN static_web_server::error_page: method=GET status=404 error=404
```
tells which page was not found.

**Comment by joseluisq** [comment]
Now the server will show the uri when there is a 404.

```sh
static-web-server -p 8787 -d docker/public/ -g warn

# 2022-05-08T20:39:57.864548Z  WARN static_web_server::error_page: method=GET uri=/assets/abcdef?sort=1 status=404 error="Not Found"
# 2022-05-08T20:44:04.214944Z  WARN static_web_server::error_page: method=GET uri=/assets/abc status=404 error="Not Found"
```

**Comment by joseluisq** [comment]
Released on [v2.9.0](https://github.com/joseluisq/static-web-server/releases/tag/v2.9.0).
